src/uu/ls/src/ls.rs

```diff
@@ -2694,6 +2711,14 @@ fn display_item_long(
     quoted: bool,
 ) -> UResult<()> {
     let mut output_display: String = String::new();
+
+    // apply normal color to non filename outputs
+    if let Some(color) = &config.color {
```

Comment:
looks like it is the same code as above, maybe create a function ?

Reply:
yes, that would be a good idea